apple-pie/index.html

```diff
@@ -5,17 +5,59 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
     <!-- don't forget to link your styles -->
-  </head>
+    <link rel="stylesheet" href="./styles/style.css"> 
+
+     </head>
+
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
+   <section id = section1> 
+   
+   <div class = "centered"><header>Apple Pie</header></div>
+   
+    
+    <div class ="recipe-story"><p>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
     be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! Ingredients 1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter 3 tablespoons
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
+    favorites as well!</p> </div>
+ <div> <img class="bar" src = "images/cooking-info.png" alt="image cooking"></div>
+
+ <div><hr></div> 
+ 
+  </section>
+   
+    <section id = section2>
+
+   <div class="header"> <h2>Ingredients</h2> </div>
+    
+   <div> <ul class="dashed">
+      <li>1 recipe pastry for a 9 inch double crust pie</li>
+      <li> 1/2 cup unsalted butter </li>
+      <li>3 tablespoons all-purpose flour</li>
+      <li> 1/4 cup water</li>
+       <li>3 tablespoons all-purpose flour</li>
+        <li>1/4 cup water</li>
+        <li> 1/2 cup white sugar</li> 
+    <li>1/2 cup packed</li>
+    <li>brown sugar 8 Granny Smith apples - peeled, cored and sliced </li>
+    </ul></div>
+<div>
+
+  <img class="bar1" src ="images/recipe-info.png" alt="recipe"></div>
+    <div><hr></div>
+   </section>
+   
+   <section id = section3>
+ <div><h2>Directions</h2> </div>
+   
+  <div> <ol><li> oven to 425 degrees F (220 degrees
```

Comment:
code should be nested by tags
```suggestion
<div> 
    <ol> 
        <li> oven to 425 degrees F (220 degrees
```